api/src/services/assets.ts

```diff
@@ -38,8 +38,8 @@ export class AssetsService {
 	constructor(options: AbstractServiceOptions) {
 		this.knex = options.knex || getDatabase();
 		this.accountability = options.accountability || null;
-		this.filesService = new FilesService(options);
 		this.authorizationService = new AuthorizationService(options);
+		this.filesService = new FilesService({ ...options, accountability: undefined });
```

Comment:
This is the potentially risky change, but the auth check on line 69 should mean this file service doesn't have to be authenticated